views/utils.njk

```diff
@@ -0,0 +1,9 @@
+{% macro pascalToString(value) %}
```

Comment:
Do we have a unit test for this logic?

Reply:
This is no longer required is it?

Reply:
Not now it's being handled in the controller.